apps/frontend/src/app/5_pages/EarnPage/EarnPage.tsx

```diff
@@ -438,6 +441,24 @@ const EarnPage: FC = () => {
             />
           )}
 
+          <SimpleTable className="mt-3">
+            <SimpleTableRow
+              label={
+                <span className="relative">
```

Comment:
`HelperButton` is not vertically centered, use divs and flexbox.

<img width="290" alt="image" src="https://user-images.githubusercontent.com/14878713/234589726-5f2716c9-942e-4daf-b86a-42ccd9b70479.png">